### DISCUSSION TITLE
Expand tools

### DISCUSSION THREAD

**Discussion by lsalamon** [muttleyxd/clang-tools-static-binaries#27] [General]
How expand files using only command line terminal?

**Comment by Ballen7** [comment]
I dont understand your question. Do you mean how to decompress?

**Reply by lsalamon** [reply]
Yes. I need only clang-tidy to try test on Oracle Linux. And have only terminal to do.

**Reply by Ballen7** [reply]
I am on Mac ARM so I cannot test this myself but I believe you can download the exact binary you want. You do not need to download the compressed folder with all of them... e.g.

`wget https://github.com/muttleyxd/clang-tools-static-binaries/releases/download/master-1d7ec53d/clang-tidy-15_linux-amd64`

then when its done downloading do

`sudo chmod +x clang-tidy-15_linux-amd64`

and see if it works
